prijateli_tree/app/templates/base.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta http-equiv="content-language" content="en-us">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
+    {% block meta %}
+      <meta name="description"
+            content="A FastAPI application used to play lab games and collect survey data.">
+    {% endblock meta %}
+    <title>
+      {% block title %}
+        Prijateli Tree
+      {% endblock title %}
+    </title>
+    <link href="{{ url_for('static', path='/mui.min.css') }}" rel="stylesheet">
+    <link href="{{ url_for('static', path='/mui.min.js') }}">
+  </head>
+  <body role="document">
+    {% block content %}
+    {% endblock content %}
```

Comment:
@FedericoDM, the HTML files you make should go here. Use the `CSS/JS` part of this page for your styling: https://www.muicss.com/docs/v1/css-js/boilerplate-html